src/scripts/components/stories/pilot-story/components/nav-drawer/nav-drawer.tsx

```diff
@@ -120,10 +120,13 @@ const NavDrawer: FunctionComponent = () => {
                 className={styles.navLinks}
               />
               <Button
-                link={'/'}
+                link={
+                  'https://storage.googleapis.com/esa-cfs-versions/web/pilot-story/index.html#/?globe=SI-11.91I30.32I29440615I0.00I1514764800000I0Igreenhouse.xch4I'
+                }
```

Comment:
Should be enough to link to  `/?globe=SI-11.91I30.32I29440615I0.00I1514764800000I0Igreenhouse.xch4I`

Reply:
the more you know... 😅